op-node/rollup/finality/finalizer_test.go

```diff
@@ -190,8 +199,18 @@ func TestEngineQueue_Finalize(t *testing.T) {
 		l1F.ExpectL1BlockRefByNumber(refD.Number, refD, nil)
 		l1F.ExpectL1BlockRefByNumber(refD.Number, refD, nil)
 
+		l2F := &testutils.MockL2Client{}
+		defer l2F.AssertExpectations(t)
+		helperL2BlockRefByNumberWithTimes(t, l2F, 1, refA1, refB0, refB1, refC0, refC1)
+
 		emitter := &testutils.MockEmitter{}
-		fi := NewFinalizer(context.Background(), logger, &rollup.Config{}, l1F, emitter)
+		fi := NewFinalizer(context.Background(), logger, &rollup.Config{BabylonConfig: babylonCfg}, l1F, l2F, emitter)
+
+		ctl := gomock.NewController(t)
+		defer ctl.Finish()
+		sdkClient := mocks.NewMockISdkClient(ctl)
+		fi.babylonFinalityClient = sdkClient
+		helperSdkClientQueryBlockRangeBabylonFinalized(t, sdkClient, 1, refA1, refB0, refB1, refC0, refC1)
```

Comment:
same for this. let's think of a better name

Reply:
name it `mockQueryBlockRangeBabylonFinalizedWithTimes`?